### ISSUE TITLE
Object #<Stream> has no method 'resume' in Gulp 4

### ISSUE THREAD

**Issue by demisx** [ivogabe/gulp-typescript#48]
I've followed example in "Incremental Compilation", but this piece of wrapped code:

```js
gulp.task('scripts', function () {
...
  return eventStream.merge(
    tsResult.dts.pipe(gulp.dest(paths.buildDir)),
    tsResult.js.pipe(gulp.dest(paths.buildDir))
   );
}
```

throws the following error in Gulp 4:

```
/node_modules/gulp/node_modules/undertaker/node_modules/bach/node_modules/async-done/node_modules/stream-exhaust/index.js:11
  stream._read ? stream.pipe(new Sink) : stream.resume();
                                                ^
TypeError: Object #<Stream> has no method 'resume'
    at resumer (/Users/dmoore/projects/beocommedia/beotracker-com-desktop/node_modules/gulp/node_modules/undertaker/node_modules/bach/node_modules/async-done/node_modules/stream-exhaust/index.js:11:49)
    at asyncRunner (/Users/dmoore/projects/beocommedia/beotracker-com-desktop/node_modules/gulp/node_modules/undertaker/node_modules/bach/node_modules/async-done/index.js:43:11)
    at process._tickDomainCallback (node.js:463:13)
    at Function.Module.runMain (module.js:499:11)
    at startup (node.js:119:16)
    at node.js:906:3
```

**Comment by phated** [comment]
Trying to land a patch so this won't crash anymore.  `event-stream` is not going to be supported because they don't return compliant streams.  Try switching to https://www.npmjs.com/package/merge2

**Comment by phated** [comment]
Fix landed in stream-exhaust 1.0.1, async-done should no longer explode.

ref https://github.com/chrisdickinson/stream-exhaust/pull/2

The stream won't be exhausted, so make sure to pipe it to a sink or switch to merge2

**Comment by ivogabe** [comment]
Thanks! I've updated the readme (0019d7bf468d0a8f67ce38af685d77f186fd5ae9)
